src/Runtime/Fork/IPC.php

```diff
@@ -51,6 +51,7 @@ public static function create(): self
      */
     public function put(string $data): void
     {
+        // @codeCoverageIgnoreStart
```

Comment:
Same goes here, I believe this coverage is not detected on the child process, therefore Ignore it.